Hestia/Hestia/backend/models/deserializers/DeviceDeserializer.cs

```diff
@@ -14,7 +14,7 @@ public Device Deserialize(JToken jT, NetworkHandler networkHandler)
             // get id, name and type
             string id = jT.Value<string>("deviceId");
             string name = jT.Value<string>("name");
-            string type = jT.Value<string>("name");
+            string type = jT.Value<string>("type");
```

Comment:
Did this have impact for the front end?

Reply:
Yes, the type always had the same value as the name.